politeiawww/cmd/politeiawwwcli/commands/help.go

```diff
@@ -122,6 +122,8 @@ func (cmd *HelpCmd) Execute(args []string) error {
 		fmt.Printf("%s\n", setInvoiceStatusHelpMsg)
 	case "invoicecomments":
 		fmt.Printf("%s\n", invoiceCommentsHelpMsg)
+	case "batchproposals":
+		fmt.Printf("%s\n", batchProposalsHelpString)
```

Comment:
Minor nit.  Can change this variable name to `batchProposalsHelpMsg` so that its consistent with all the other ones?